solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/psc/google-apis/dns.yaml

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Private DNS zone for googleapis.com
+# SC-22
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSManagedZone
+metadata:
+  name: host-project-id-internet-googleapis-dns # kpt-set: ${host-project-id}-internet-googleapis-dns
+  namespace: client-name-networking # kpt-set: ${client-name}-networking
+  annotations:
+    cnrm.cloud.google.com/project-id: host-project-id # kpt-set: ${host-project-id}
+    config.kubernetes.io/depends-on: compute.cnrm.cloud.google.com/namespaces/client-name-networking/ComputeNetwork/host-project-id-global-internet-vpc # kpt-set: compute.cnrm.cloud.google.com/namespaces/${client-name}-networking/ComputeNetwork/${host-project-id}-global-internet-vpc
+spec:
+  description: "Private DNS zone for googleapis.com"
+  resourceID: internet-googleapis-dns
+  dnsName: "googleapis.com."
+  visibility: private
+  privateVisibilityConfig:
+    networks:
+      - networkRef:
+          name: host-project-id-global-internet-vpc # kpt-set: ${host-project-id}-global-internet-vpc
+---
+# Record Set for googleapis.com in googleapis.com DNS zone
+# SC-22
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: host-project-id-internet-googleapis-rset # kpt-set: ${host-project-id}-internet-googleapis-rset
+  namespace: client-name-networking # kpt-set: ${client-name}-networking
+  annotations:
+    cnrm.cloud.google.com/project-id: host-project-id # kpt-set: ${host-project-id}
+    config.kubernetes.io/depends-on: compute.cnrm.cloud.google.com/namespaces/client-name-networking/ComputeNetwork/host-project-id-global-internet-vpc # kpt-set: compute.cnrm.cloud.google.com/namespaces/${client-name}-networking/ComputeNetwork/${host-project-id}-global-internet-vpc
+spec:
+  name: "googleapis.com."
+  type: "A"
+  ttl: 300
+  managedZoneRef:
+    name: host-project-id-internet-googleapis-dns # kpt-set: ${host-project-id}-internet-googleapis-dns
+  rrdatas:
```

Comment:
you could replace this rrdatas to use
rrdatasRefs:
- kind: ComputeAddress
  name: host-project-id-internet-psc-apis-ip # kpt-set: ${host-project-id}-internet-psc-apis-ip

Reply:
see https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/377#discussion_r1211572462

Reply:
yeah, I agree with the suggested solution